src/app/cube/details/action-panel/action-panel.component.html

```diff
@@ -19,6 +19,11 @@
         <i class="fal fa-spinner-third fa-spin"></i>
       </span>
     </button>
+
+    <button (click)="clark.center" class="button neutral on-white" id="edit-button" type="submit"> 
```

Comment:
The next step we'll need to take here is to only show this button if the user's AuthGroup is editor or admin.